plugins/sosreport/skills/ovs-db-analysis/scripts/analyze_ovs_db.py

```
@@ -0,0 +1,1296 @@
+    if path.suffix in ['.xz', '.gz'] or '.tar' in path.name:
+        extract_dir = path.parent / f"{path.stem}_extracted"
+        
+        if not extract_dir.exists():
+            print(f"Extracting {path}...")
+            extract_dir.mkdir(parents=True)
+            
+            if path.suffix == '.xz':
+                import lzma
+                with lzma.open(path, 'rb') as xz:
+                    with tarfile.open(fileobj=xz) as tar:
+                        tar.extractall(extract_dir)
```

Ruff (0.14.8)

Uses of `tarfile.extractall()`

(S202)